src/MissionManager/PlanMasterController.cc

```diff
@@ -620,6 +631,13 @@ QStringList PlanMasterController::saveNameFilters(void) const
     return filters;
 }
 
+QStringList PlanMasterController::getAllPlans() const
+{
+      QStringList allPlans = qgcApp()->getDataClass()->getAllPlans();
+      allPlans.push_front("No Plan Selected");
```

Comment:
In general in your logs, avoid unnecessary capitalization of common words. Like this one should be "No plan selected"

Reply:
Fixed